Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -100,13 +113,13 @@ private void UpdateRaycast()
             // Check if the currently hit object has changed
             if (oldFocusedObject != FocusedObject)
             {
-                if (oldFocusedObject != null)
+                if (oldFocusedObject != null && OnGazeExit != null)
                 {
-                    oldFocusedObject.SendMessage("OnGazeLeave", SendMessageOptions.DontRequireReceiver);
```

Comment:
@NeerajW @jwittner any thoughts on this ? don't think it's the perfect excuse to at least put a release notes system in place?